app/controllers/api/v2/override_values_controller.rb

```diff
@@ -11,6 +12,8 @@ class OverrideValuesController < V2::BaseController
       before_action :return_if_smart_mismatch, :only => [:index, :create, :show, :update, :destroy]
       before_action :return_if_override_mismatch, :only => [:show, :update, :destroy]
 
+      deprecate_param [:create, :update], [:override_value, :use_puppet_default], :skip_foreman
```

Comment:
There are already a few places that do that without `deprecate_param`.
Maybe move the deprecation to a separate pr and use it in more than one place?